src/appshell/view/dockwindow/dockpanel.cpp

```diff
@@ -141,14 +136,13 @@ void DockPanel::componentComplete()
         return;
     }
 
-    m_menuModel->load();
-
     w->setProperty(DOCK_PANEL_PROPERY, QVariant::fromValue(this));
-    w->setProperty(CONTEXT_MENU_MODEL_PROPERTY, m_menuModel->toVariant());
 
-    connect(this, &DockPanel::contextMenuModelChanged, [this, w]() {
+    m_menuModel->load();
+
+    connect(m_menuModel, &AbstractMenuModel::itemsChanged, [w, this]() {
         if (w) {
```

Comment:
Please, rename it to dockWidget

Reply:
renamed, see
https://github.com/musescore/MuseScore/pull/9006